great_expectations/checkpoint/util.py

```diff
@@ -44,23 +44,17 @@ def send_slack_notification(query, slack_webhook=None, slack_channel=None, slack
 
     try:
         response = session.post(url=url, headers=headers, json=query)
-        if slack_webhook:
-            ok_status = response.text == "ok"
```

Comment:
I'm assuming these ok checks were just redundant because we always got okay for 200 responses?

Reply:
yup I believe so - I've made the checks a bit more robust with `raise_for_status`